core/src/main/java/org/eclipse/hono/util/RegistrationConstants.java

```diff
@@ -71,6 +71,14 @@
      */
     public static final String FIELD_MEMBER_OF = "memberOf";
 
+    /**
+     * The name of the field in a device registration information that contains the identifier of the names
+     * of the groups in which the device is member of (either as string value or inside a JSON array).
+     * Note, that it is only possible to either set the 'via' or the 'memberOf' property since groups of groups
+     * are currently not supported.
```

Comment:
This seems to be the description of the *memberOf* field ...

Reply:
right, forgot to change it.